### PR TITLE
fix: #48 implement semantic version

### PR DESCRIPTION
**Pull request by DiegoTinitana**

### CODE REVIEW

File: .github/workflows/Release-time-tracker-ui.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
```
Comment (macrisguncay) [inline .github/workflows/Release-time-tracker-ui.yml:16]: node version should be " node-version: '12.x' "

File: .github/workflows/Release-time-tracker-ui.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Release
```
Comment (enriquezrene) [inline .github/workflows/Release-time-tracker-ui.yml:1]: let's change it to:
```
bumping-versions-pipeline
```

File: .github/workflows/Release-time-tracker-ui.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
```
Comment (enriquezrene) [inline .github/workflows/Release-time-tracker-ui.yml:7]: bumping-version

File: README.md [hunk 4]
```diff
@@ -52,6 +52,15 @@ Install the following extensions:
 
 ## Code scaffolding
 
+## Commit format
```
Comment (macrisguncay) [inline README.md:55]: Your text block is hurting the "code scaffolding" section please solve it.
I think your text block would fit better as another option of the menu "Prepare your environment". 
Please change "Commit format" to "Commit message format"

File: README.md [hunk 5]
```diff
@@ -52,6 +52,15 @@ Install the following extensions:
...
+  We are using format by release and tags: https://semver.org/
```
Comment (enriquezrene) [inline README.md:56]: A commit message needs to start with one of the following words to bump the application version properly (This application is following a semver strategy for versioning https://semver.org/)

File: README.md [hunk 6]
```diff
@@ -52,6 +52,15 @@ Install the following extensions:
 
 ## Code scaffolding
 
+## Commit format
+  We are using format by release and tags: https://semver.org/ 
+  ### Sumary
+  - fix is equal to Patch Release example: 1.0.1
+  - feat is equal to Feature Release example: 1.1.0
+  - perf is equal to Breaking Release example: 2.0.0
```
Comment (enriquezrene) [inline README.md:60]: Let's mark the keywords in bold:

  - **fix** is equal to Patch Release example: 1.0.1
  - **feat** is equal to Feature Release example: 1.1.0
  - **perf** is equal to Breaking Release example: 2.0.0

File: package.json [hunk 7]
```diff
@@ -67,8 +72,42 @@
   },
   "config": {
     "commit-message-validator": {
-      "pattern": "^(#([0-9]+).*)",
-      "errorMessage": "Your commit message must start with issue number e.g. #39"
+      "pattern": "^(fix: #|feat: #|perf: #)[0-9].*",
+      "errorMessage": "Your commit should has this format> fix or feat or perf: #45 message"
```
Comment (enriquezrene) [inline package.json:76]: Your commit message needs to start with fix:, feat:, or perf: followed by issue number, e.g. fix: #43 any commit message

**Pull request merged**